app/controllers/coronavirus_landing_page_controller.rb

```diff
@@ -17,4 +17,16 @@ def content_item
   def presenter
     CoronavirusLandingPagePresenter.new(content_item.to_hash)
   end
+
+  def cookie_policy
+    if cookies.has_key?("cookies_policy")
+      JSON.parse(cookies["cookies_policy"])
+    else
+      {}
+    end
+  end
+
+  def campaign_cookie
+    cookie_policy["campaigns"]
```

Comment:
Won't this fail if the above returns {}

Reply:
it will return nil which is ok

Reply:
In the `_live_stream` partial you'd have 
```
<% if campaign_cookie == true %>
 show video
<% else %>
show placeholder
<% end %>